etc/development.ini

```diff
@@ -3,11 +3,13 @@ pipeline =
     conduit
 
 [app:conduit]
-use = egg:conduit
+use = config:production.ini#conduit
 
-pyramid.debug_authorization = false
-pyramid.debug_notfound = false
-pyramid.debug_routematch = false
```

Comment:
^^ These three should be set to `true`

Reply:
Originally these were `false` in both production and development. 
Since development now inherits from production, I removed them.
But it was probably a mistake ?

I'll change.

Reply:
Done.

Reply:
Hmm, I wonder why I set them to `false`? Let's keep them, I guess I had a good reason.

Sorry for the noise!

Reply:
I'll just remove the latest commit and force-push 😄